### DISCUSSION TITLE
rook-discover-pod's log becomes bigger.

### DISCUSSION THREAD

**Discussion by mc956759** [rook/rook#12318] [Q&A]
After my ceph cluster running 1 month, I found that one rook-discover-pod's log size reached to 74GB.  like this:
“
[root@rook-discover-nx69w log]# ls -rhtl ceph
**-rw-r--r-- 1 root root 74G Jun  1 10:33 ceph**
[root@rook-discover-nx69w log]# pwd
/var/log
”
I want to know how to limit the log size? And how can I reduce the current log size?
btw. do i have to run the rook-discover component? If I specify osd-devices in the cluster.yaml.
Thanks for any help!

**Comment by travisn** [comment]
If you restart the discover daemon, does that path get reset? Or it just keeps growing?

The discover daemon is disabled by default in new deployments for many releases, and can be disabled with [this setting](https://github.com/rook/rook/blob/release-1.11/deploy/examples/operator.yaml#L493-L495). That link describes the scenario where you might want to keep it enabled.
